platform/qt/src/local_glyph_rasterizer.cpp

```diff
@@ -68,8 +72,13 @@ Glyph LocalGlyphRasterizer::rasterizeGlyph(const FontStack&, GlyphID glyphID) {
     // Render at constant baseline, to align with glyphs that are rendered by node-fontnik.
     painter.drawText(QPointF(0, 20), QString(QChar(glyphID)));
 
+#if QT_VERSION >= QT_VERSION_CHECK(5, 10, 0)
+    auto img = std::make_unique<uint8_t[]>(image.sizeInBytes());
+    memcpy(img.get(), image.constBits(), image.sizeInBytes());
+#else
     auto img = std::make_unique<uint8_t[]>(image.byteCount());
     memcpy(img.get(), image.constBits(), image.byteCount());
+#endif
```

Comment:
Maybe just use variable to get sizeInBytes/byteCount in version-specific if-else directive and then use that variable  in make_unique/memcpy so that statements are not repeated?

Reply:
Sorry, I'm very busy this week. Let me follow-up in one of my next MRs.

Reply:
No prob, this was just a "cosmetic" issue. Thanks for your help!